test/unit/foreman/renderer_test.rb

```diff
@@ -221,4 +221,8 @@ def setup_safemode_renderer
     @renderer.instance_variable_set '@whatever_random_name', 'has_value'
     assert_equal({ :whatever_random_name => 'has_value' }, @renderer.send(:allowed_variables_mapping, [ :whatever_random_name ]))
   end
+
+  test 'templates_used is allowed to render for host' do
+    assert Safemode.find_jail_class(Host::Managed).allowed? :templates_used
```

Comment:
I think this test only verifies that safemode jail can list allowed methods. I agree it's not required to test the output but I'd like to have a test that verifies it can render without errors at least. Renderer is sort of a user API so I'd like to have it covered.

you could use something like this in block near line 151
```ruby
    test "#{renderer_name} should render templates used" do
      send "setup_#{renderer_name}"
      @renderer.host = FactoryGirl.build(:host)
      template = mock('template')
      template.stubs(:template).returns('<%= @host.templates_used %>')
      assert_nothing_raised do
        @renderer.unattended_render(template)
      end
    end
```